examples/verifying_gfd/verifying_gfd1.py

```diff
@@ -0,0 +1,101 @@
+from pathlib import Path
+
+import desbordante
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+
+
+class bcolors:
+    HEADER = '\033[95m'
+    WARNING = '\033[93m'
+    ENDC = '\033[0m'
+
+
+GRAPH_NAME = 'blogs_graph'
+MODIFIED_GRAPH_NAME = 'incorrect_author_blogs_graph'
+GFD_NAME = 'correct_author_gfd'
+
+GRAPHS_DATASETS_FOLDER_PATH = 'examples/verifying_gfd/datasets/graphs'
+GFDS_DATASETS_FOLDER_PATH = 'examples/verifying_gfd/datasets/gfds'
+
+GRAPH = Path(f'{GRAPHS_DATASETS_FOLDER_PATH}/{GRAPH_NAME}.dot')
+MODIFIED_GRAPH = Path(f'{GRAPHS_DATASETS_FOLDER_PATH}/{MODIFIED_GRAPH_NAME}.dot')
+GFD = Path(f'{GFDS_DATASETS_FOLDER_PATH}/{GFD_NAME}.dot')
+
+GRAPHS_FIGURES_FOLDER_PATH = 'examples/verifying_gfd/figures/graphs'
+GFDS_FIGURES_FOLDER_PATH = 'examples/verifying_gfd/figures/gfds'
+
+GRAPH_IMAGE = Path(f'{GRAPHS_FIGURES_FOLDER_PATH}/{GRAPH_NAME}.png')
+MODIFIED_GRAPH_IMAGE = Path(f'{GRAPHS_FIGURES_FOLDER_PATH}/{MODIFIED_GRAPH_NAME}.png')
+GFD_IMAGE = Path(f'{GFDS_FIGURES_FOLDER_PATH}/{GFD_NAME}.png')
+
+GRAPH_INFO = ('The graph is depicted in figure. The following abbreviations '
+              'were used: A - account, B - blog. Vertices labeled A have a '
+              '"name" attribute showing the nickname; vertices labeled B - '
+              '"author", indicating who wrote the blog. The values of these '
+              'attributes are labeled next to the vertices. The edges are also '
+              'labeled as: "post", which indicates who wrote the blog, and '
+              '"like", which indicates approval by another person. In the '
+              'drawing, the edges are marked "post" in bold.\n')
+
+GFD_INFO = ('If the graph functional dependency on a figure is not satisfied, '
+              'the data are erroneous because the information contained in the '
+              'edge label contradicts the information about the author contained '
+              'at the vertex of the blog.\n')
```

Comment:
```suggestion
            'the data are erroneous because the information contained in the '
            'edge label contradicts the information about the author contained '
            'at the vertex of the blog.\n')
```
Continuation lines should align wrapped elements either vertically using Python’s implicit line joining inside parentheses, brackets and braces... (see [here](https://peps.python.org/pep-0008/#indentation))
Same for the other occurrences.